.github/workflows/coding-standards.yml

```diff
@@ -1,14 +1,36 @@
 name: "Coding Standards"
 
 on:
-  pull_request:
-    branches:
-      - "*.x"
   push:
-    branches:
-      - "*.x"
 
 jobs:
   coding-standards:
     name: "Coding Standards"
-    uses: "doctrine/.github/.github/workflows/coding-standards.yml@5.0.1"
```

Comment:
Why not reference your fork instead of copy/pasting everything?

Reply:
I reused CI file I used for private development... Now I demonstrated the function using the fork/PR branch.